pkg/file/splitter/internal/job.go

```diff
@@ -47,14 +48,15 @@ type SimpleSplitterJob struct {
 	cursors    []int    // section write position, indexed per level
 	hasher     bmt.Hash // underlying hasher used for hashing the tree
 	buffer     []byte   // keeps data and hashes, indexed by cursors
-	toEncrypt  bool     // to encryrpt the chunks or not
+	tagg       *tags.Tag
+	toEncrypt  bool // to encryrpt the chunks or not
```

Comment:
afaiu encryption is not currently being performed. Please let us follow our existing policy of not including something that "perhaps" will be implemented in the future.

in any case, I think embedding encryption into this component is an architecturally weak decision.